guides/source/ruby_on_rails_guides_guidelines.md

```diff
@@ -64,7 +64,7 @@ The guides and the API should be coherent and consistent where appropriate. In p
 HTML Guides
 -----------
 
-Before generating the guides, make sure that you have the latest version of Bundler installed on your system. As of this writing, you must install Bundler 1.3.5 on your device.
+Before generating the guides, make sure that you have the latest version of Bundler installed on your system. As of this writing, you must install Bundler 1.3.5 or later on your device.
```

Comment:
It's not that it didn't work, but I think the way it is written now it kinda suggests that only v1.3.5 should be used.